jjb/edgex-macros.yaml

```diff
@@ -1,17 +1,9 @@
 ---
-
-- property:
-    name: 'edgex-go-x86_64'
-    properties:
-      - inject:
-          properties-content: |
-            GOPATH=$WORKSPACE
-
-- property:
-    name: 'edgex-go-amd64'
-    properties:
-      - inject:
-          properties-content: |
-            GOPATH=$WORKSPACE
-            GOARCH=amd64
-
+- builder:
+    name: lf-provide-docker-cleanup
```

Comment:
We should not be adding things to the lf- macro / template namespace inside projects. That is reserved for global-jjb use only. Instead, if you want to be safe and namespace this, use edgex-